### ISSUE TITLE
Global DNS UI: Unable to select multi cluster app if a project was selected

### ISSUE THREAD

**Issue by davidnuzik** [rancher/rancher#17549]
Version: 2.2 master (1/22/19) HA Setup

**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
- Create a Global DNS entry pick the "Projects" radio button and pick on of the projects under the Target Projects list.
![image](https://user-images.githubusercontent.com/45179589/51637514-42fc8680-1f19-11e9-9fa7-8fab6079e8b7.png)
- Now, click on the "Multi Cluster App" radio button so you swich context to picking an app. Notice this is grayed out.
![image](https://user-images.githubusercontent.com/45179589/51637594-6de6da80-1f19-11e9-8dbc-e1d901b0ced4.png)
- Go back to the "Projects" radio button and unselect that project. Go back to "Multi Cluster App" again and now the drop-down is no longer grayed out.
![image](https://user-images.githubusercontent.com/45179589/51637695-b30b0c80-1f19-11e9-9f08-2d7a313955da.png)

**Result:**
You should be able to pick a Multi Cluster App even if you had picked a target project(s) under the "Projects" radio button.

**Other details that may be helpful:**
It seems as though the UI still thinks we have a project selected even though we no longer can see our list of target projects once we pick "Multi Cluster App" radio button.

It may also be worth noting, that if you delete the projects that we had associated with the DNS entry (so there are no longer any targets) the "Multi Cluster App Target" dropdown is still grayed out. To resolve, you also go back to "Projects" radio button pick a project and un-pick it and go back to "Multi Cluster App" radio button and now the drop down is available again.

**Comment by cjellick** [comment]
@westlywright backend is merged see https://github.com/rancher/rancher/issues/18366#issuecomment-467232466

**Comment by sangeethah** [comment]
@westlywright  There was another change in api done -
https://github.com/rancher/rancher/issues/18366#issuecomment-469385830

**Comment by westlywright** [comment]
PR https://github.com/rancher/ui/pull/2723 addresses this

**Comment by davidnuzik** [comment]
Version: Master 2.2 (3/5/19)
Confirmed issue is resolved. Followed the steps to reproduce and when I test now I can see that I can pick a Multi Cluster App after I had already picked projects under the Projects radio button.

When I switch back to Multi-Cluster app it is no longer grayed-out when I have selected projects.
![image](https://user-images.githubusercontent.com/45179589/53899910-7e01d780-3ff8-11e9-9829-1da8bb6c6609.png)

When I switch back to projects selection I can see my previously-selected projects are now de-selected, which is expected behavior.
![image](https://user-images.githubusercontent.com/45179589/53899927-8eb24d80-3ff8-11e9-9846-5c94bec1de53.png)

I smoke tested this area. I created pointing to projects, save, edit again, switch to multicluster app, and vice versa. Each time this is working as expected now.
